### PR TITLE
Fixes FuncRef global initialization with imported globals 

### PR DESCRIPTION
**Pull request by mathetake**
A bug was found in #886 where if a module imports a FuncRef type global and uses it as an initial value for a locally defined FuncRef global.

### CODE REVIEW

File: internal/integration_test/fuzzcases/fuzzcases_test.go [hunk 1]
```diff
@@ -343,3 +345,38 @@ func Test874(t *testing.T) {
 		require.NoError(t, err)
 	})
 }
+
+func Test888(t *testing.T) {
```
Comment (codefromthecrypt) [inline internal/integration_test/fuzzcases/fuzzcases_test.go:349]: lucky number

**Review by codefromthecrypt — APPROVED** [review]
good when fixing something makes it simpler

File: internal/integration_test/fuzzcases/fuzzcases_test.go (hunk 1 above)
Comment (mathetake) [reply internal/integration_test/fuzzcases/fuzzcases_test.go:349]: in japan lucky number is 777

Comment (codefromthecrypt) [reply internal/integration_test/fuzzcases/fuzzcases_test.go:349]: off by 111 problem

**Pull request merged**